src/Forms/FormField.php

```diff
@@ -227,6 +233,27 @@ public function getNameRaw()
         return $this->name;
     }
 
+    public function getAutoComplete()
+    {
+        return $this->autoCompleteComponent;
+    }
+
+    /**
+     * @param $component
+     * @return $this
+     * @throws \Exception
+     */
+    public function setAutoComplete($component)
+    {
+        if (!isset(AutoCompleteConstants::COMPONENTS[$component])) {
```

Comment:
If you're going to force exceptions, you need to make sure ALL possible options exist, like "on" and "off"

Reply:
Added ON/OFF options!